packages/serverless-orchestration/package.json

```diff
@@ -12,8 +12,7 @@
     "@awaitjs/express": "^0.3.0",
     "@google-cloud/datastore": "^6.0.0",
     "@google-cloud/storage": "^2.4.2",
-    "@openzeppelin/contracts": "3.0.0",
-    "@uma/core": "npm:@uma/core@1.2.0",
+    "@uma/core": "^2.2.0",
```

Comment:
Did you mean to do this upgrade in this PR?

Reply:
yup Previously this was bound to this old core version. this meant that the tests in this package broke due to the version bumping.